static/js/around-map.js

```diff
@@ -33,7 +37,7 @@ worker.addEventListener('message', response => {
 
     if(window.location.hash) {
```

Comment:
Well yes, but I did not use it right after (`if(hash === el.id) {`).
Here we're talking about code refactoring. What I did + what you're doing = new baby. 👶